docs/timeline/index.html

```diff
@@ -944,6 +944,14 @@ <h2 id="Configuration_Options">Configuration Options</h2>
     </tr>
 
     <tr>
+      <td>onInitialDrawComplete</td>
```

Comment:
Small niggle on my part: The options are arranged alphabetically, this new entry and `onUpdate` are not.